Samples/Samples.Cards/appsettings.Development.json

```diff
@@ -12,5 +12,6 @@
   "Teams": {
     "ClientId": "",
     "ClientSecret": ""
-  }
+  },
+  "USE_MINIMAL_API": false
```

Comment:
nit: change to `UseMinimalApi` to match pascal naming conventions of C# appsettings